tests/test_main.py

```diff
@@ -461,3 +454,48 @@ class User(BaseModel):
     u1 = User()
     u2 = User()
     assert u1.friends is not u2.friends
+
+
+class ArbitraryType:
+    pass
+
+
+def test_arbitrary_type_allowed_validation_success():
+    class ArbitraryTypeAllowedModel(BaseModel):
+        t: ArbitraryType
+
+        class Config:
+            arbitrary_types_allowed = True
+
+    arbitrary_type_instance = ArbitraryType()
+    m = ArbitraryTypeAllowedModel(t=arbitrary_type_instance)
+    assert m.t == arbitrary_type_instance
+
+
+def test_arbitrary_type_allowed_validation_fails():
+    class ArbitraryTypeAllowedModel(BaseModel):
+        t: ArbitraryType
+
+        class Config:
+            arbitrary_types_allowed = True
+
+    class C:
+        pass
+
+    with pytest.raises(ValidationError) as exc_info:
+        ArbitraryTypeAllowedModel(t=C())
+    assert exc_info.value.errors() == [
```

Comment:
can you please test your changes with `exc_info.value.json()`? I don't think that `ctx` with type is JSON serializable. To my mind we need to use `utils.display_as_type` when we firing `ArbitraryTypeError` exception.